### ISSUE TITLE
7.4.1. Screen Name/Title: Browse templates – Empty state - Copy template - Search by name (Optional)

### ISSUE THREAD

**Issue by whabanks** [cds-snc/notification-planning#2319]
**Component:** 
Copy an existing template - Search by name (Optional)

**Observation:**
The <input> element for search is missing an accessible name. Although it is associated with a <label>, there is no explicit aria-labelledby or aria-label to ensure screen readers can interpret it reliably.

**Recommendation:** 
Ensure the `<input>` element is properly associated with the `<label>` by checking that the ‘for’ attribute of the label exactly matches the id of the input (id="search" in this case). Also, avoid having empty `<span></span>` between the label and input, which may confuse screen readers.

**Comment by smcmurtry** [comment]
not an issue
